### DISCUSSION TITLE
Where should the examples folder go in the spawning chapter

### DISCUSSION THREAD

**Discussion by GithubSre** [tokio-rs/tokio#2986] [Q&A] [answered]
it is not working if I say mv src/main.rs examples/hello-redis.rs
I am not even understanding where to create this folder mkdir -p examples
I can't move examples folder to the onther folder.
where should i create examples folder please tell me.
not understanding

**Comment by Darksonn** [comment, marked as answer]
I'm assuming you're referring to [the spawning chapter][1]. The `examples/` folder should go next to the `src/` folder.
```
my-redis
├── examples
│   └── hello-redis.rs  <-- old main.rs file
└── src
    └── main.rs         <-- new empty file
```
If this does not work, please explain how it fails. What is the error message?

[1]: https://tokio.rs/tokio/tutorial/spawning

**Reply by GithubSre** [reply]
Thaaaaaaaaaaaaaaaaaaanks very much
